view/frontend/web/template/payment/offsite-alipay-form.html

```diff
@@ -0,0 +1,41 @@
+<div class="payment-method"
```

Comment:
@jacekstan This file's indentation is mixed of space and tab, can you convert it all to space?